hierarchicalforecast/methods.py

```diff
@@ -59,23 +64,243 @@ def _reconcile(S: np.ndarray, P: np.ndarray, W: np.ndarray,
             res[f'hi-{lv}'] = res['mean'] + zs * sigmah
     return res
 
+# %% ../nbs/methods.ipynb 5
+class Normality:
```

Comment:
I've changed the name to normality since it is a general method that works independently of the reconciliation method: https://otexts.com/fpp3/rec-prob.html